### PR TITLE
miniBERTas support

### PR DESCRIPTION
**Pull request by phu-pmh**
Add support for miniBERTas: tested and working now.

### CODE REVIEW

File: jiant/proj/main/export_model.py [hunk 1]
```diff
@@ -29,8 +29,12 @@ def export_model(model_type, output_base_path, model_class, tokenizer_class):
     os.makedirs(tokenizer_fol_path, exist_ok=True)
     os.makedirs(model_fol_path, exist_ok=True)
 
-    model_path = os.path.join(model_fol_path, f"{model_type}.p")
-    model_config_path = os.path.join(model_fol_path, f"{model_type}.json")
+    # nyu_mll/roberta-base creates spurious directory
+    model_filename = model_type.replace("/", "-")
+    print(model_filename)
```
Comment (jeswan) [inline jiant/proj/main/export_model.py:34]:
```suggestion

```

File: jiant/proj/main/export_model.py [hunk 2]
```diff
@@ -29,8 +29,12 @@ def export_model(model_type, output_base_path, model_class, tokenizer_class):
     os.makedirs(tokenizer_fol_path, exist_ok=True)
     os.makedirs(model_fol_path, exist_ok=True)
 
-    model_path = os.path.join(model_fol_path, f"{model_type}.p")
-    model_config_path = os.path.join(model_fol_path, f"{model_type}.json")
+    # nyu_mll/roberta-base creates spurious directory
+    model_filename = model_type.replace("/", "-")
```
Comment (jeswan) [inline jiant/proj/main/export_model.py:33]: @zphang the model name `nyu_mll/roberta-base` contains `/` which causes a new folder to be created. Do any alternatives come to mind to handle this situation?

Comment (zphang) [reply jiant/proj/main/export_model.py:33]: Yep. This was resolved with the update from: https://github.com/jiant-dev/jiant/pull/129

How about I break that off as a standalone PR?

Comment (jeswan) [reply jiant/proj/main/export_model.py:33]: @zphang That would be great! @phu-pmh Let's adapt this PR based on those changes.

**Comment by jeswan** [comment]
This PR is superseded by https://github.com/jiant-dev/jiant/pull/147

**Pull request closed without merging**